lib/generator/characteristic.js.erb

```diff
@@ -26,12 +26,16 @@
   matching = matchingValue(value, 'true');
   matching.specificContext=hqmf.SpecificsManager.identity();
   return matching;
-<%- elsif criteria.property == :race -%>
-  matching = new Boolean(value.includedIn(<%=criteria.inline_code_list.to_json%>));
+<%- elsif criteria.property == :race || criteria.property == :payer -%>
+  if (value === null) {
+    matching = false;
```

Comment:
It looks like, in order to have an attribute like ```specificContext```, a boolean needs to be declared using ```new Boolean```, in this case ```new Boolean(false)```, rather than just using the literal. Tested using Chrome.